test/JDBC/expected/TestErrorHelperFunctions.out

```diff
@@ -39,139 +39,165 @@ int#!#varchar
 ~~END~~
 
 
-select * from fn_mapped_system_error_list();
+select pg_sql_state, sql_error_code from fn_mapped_system_error_list() order by pg_sql_state;
```

Comment:
we should do a select * from the function

Reply:
done!